resources/layout-page/src/main/resources/resources/widgets/customLivingApplicationMenu/customLivingApplicationMenu.js

```diff
@@ -14,7 +14,7 @@
     ctrl.pageToken =  pathArray[pathArray.length-2];
     
     this.isSticky = function() {
-        $window.document.body.style.paddingTop = $scope.properties.sticky ? "70px": "0px";
+        $window.document.body.style.paddingTop = $scope.properties.sticky ? "50px": "0px";
```

Comment:
Maybe (Surely ?) this should be moved to `layout.css` so it can be overridden in themes